client/jetpack-connect/index.js

```diff
@@ -135,13 +141,17 @@ export default function () {
 		);
 	}
 
-	page(
-		'/jetpack/connect/plans/:interval(yearly|monthly)?/:site',
-		siteSelection,
-		controller.plansSelection,
-		makeLayout,
-		clientRender
-	);
+	if ( shouldShowOfferResetFlow() ) {
+		plansV2( `/jetpack/connect/plans/:site`, siteSelection, controller.offerResetContext );
+	} else {
+		page(
+			'/jetpack/connect/plans/:interval(yearly|monthly)?/:site',
```

Comment:
Not to request a change, but to keep the question in mind. Wouldn't it be better if we reuse the same URL structure since users might be redirected here from different sources that we aren't planning to update?

Reply:
Yeah, should do a redirect, but also want to figure out the locale question so figured it'd be good to tackle in a future ticket. Good thought!